content/docs/ops/security/keys-and-certs/index.md

```diff
@@ -179,3 +181,87 @@ Certificate:
          0c:33:42:f0:b6:f5:1c:0d:fb:70:65:aa:93:3e:ca:0e:58:ec:
          8e:d5:d0:1e
 {{< /text >}}
+
+## How Citadel decides whether to create Service Account secrets
+
+### Overview
+
+When a Citadel instance notices that a `ServiceAccount` was created in a namespace, it needs to decide whether it should generate an `istio.io/key-and-cert` secret for that `ServiceAccount`. In order to make that decision, there are three relevant values:
+
+1. **`ca.istio.io/env` namespace label**: *string valued* label which should be pointed at the namespace of the desired Citadel instance
+
+1. **`ca.istio.io/override` namespace label**: *boolean valued* label which overrides all other configurations and forces all Citadel instances either to target or ignore a namespace
+
+1. [**`enableNamespacesByDefault` security configuration**](/docs/reference/config/installation-options/#security-options): default behavior if no labels are found on the `ServiceAccount's` namespace
+
+From these three values, the decision process mirrors that of the [`Sidecar Injection Webhook`](/docs/ops/setup/injection/). The detailed behavior is that:
+
+* If `ca.istio.io/override` exists and is valid, then follow what it tells us
+* If not, then check `ca.istio.io/env`. If the value matches the Citadel instance's namespace, the namespace should be targeted
+* If `ca.istio.io/env` is nonexistent or invalid, follow the `enableNamespacesByDefault` configuration value
+* If `enableNamespacesByDefault` is `true`, the Citadel instance should create secrets for unlabeled namespaces, otherwise it should not
+
+This logic is captured in the truth table below:
+
+| `ca.istio.io/override` value | `ca.istio.io/env` match | `enableNamespacesByDefault` configuration | Workload secret created |
```

Comment:
I think throughout above steps and `match` here, it'd be better to call out/emphasize that we may have multi citadel instance and the steps && table here are discussing behavior of a particular citadel instance.